lib/probes/express.js

```diff
@@ -128,13 +129,13 @@ function patchRoutes (express, version) {
       logMissing('v4 Route.prototype')
       return
     }
-    methods.concat('all').forEach(method => {
-      if (typeof proto[method] !== 'function') {
-        logMissing(`v4 Route.${method}()`)
+    http.METHODS.concat('all').forEach(method => {
+      if (typeof proto[method.toLowerCase()] !== 'function') {
+        logMissing(`v4 Route.${method.toLowerCase().to}()`)
```

Comment:
In `method.toLowerCase().to` what does the `.to` do?

Reply:
It was s a typo that passed tests. Fixed. Thanks.